pyproject.toml

```diff
@@ -150,6 +150,16 @@ include-package-data = true
 where = ["src"]
 
 [tool.setuptools_scm]
+tag_regex = "^v(?P<version>20\\d{2}\\.\\d{1,2}\\.\\d{1,2})$"
+git_describe_command = [
+    "git",
+    "describe",
+    "--dirty",
+    "--tags",
+    "--long",
+    "--match",
+    "v20*",
+]
```

Comment:
This changes the command which is used by `setuptools_scm` to gather information from `git` about the tags and current state of the repository. By using the `v20*` glob, it will only search for tags which match our current versioning scheme.

Reply:
We can deal with the eventual failure in 77 years :)